.rfc/RFC0000-rfc-process.md

```diff
@@ -0,0 +1,156 @@
+---
+RFC:          RFC0000
+Author:       Mikey Lombardi
+Status:       Draft
+SupercededBy: N/A
+Version:      1.0
+Area:         Process
+---
+
+# DSC RFC Process and Guidelines
+
+A DSC RFC (Request for Comments) is a publication to propose design changes and improvements to
+DSC. This provides the community an opportunity to provide feedback before code is written where it
+becomes harder to change at the risk of compatibility.
+
+This process was adapted from the PowerShell RFC process, which itself was adapted from the Chef
+RFC process and the DMTF.org process.
+
+## Roles
+
+- **Author**: All members of the community are allowed to author new RFCs and can provide feedback
+  to any RFC.
+- **DSC Committee**: The design committe that votes to accept or reject an RFC. Currently, the DSC
+  committee includes the members of the DSC team.
+- **Committee Member**: An individual member of the DSC Committee.
+- **Working Group (WG)**: A group responsible for deciding whether or not an issue in the
+  repository requires a proposal and for providing feedback within an RFC proposal.
+  
+  For more information about Working Groups, see [Working Groups][01].
+
+## Submitting an RFC
+
+When submitting an RFC, the Author shall:
+
+- Create a file named `RFCNNNN-<Title>.md` in the `.rfc/drafts` folder.
```

Comment:
Is there a reason to use `.rfc` instead of just `RFC`?  I don't see a need for it to be hidden?